drivers/entropy/entropy_nrf5.c

```diff
@@ -65,10 +67,27 @@ static int entropy_nrf5_get_entropy(struct device *device, u8_t *buf, u16_t len)
 	}
 
 	return 0;
+
+#else
+	/* ATTENTION: calling this function implies that a bt init function has
+	 * been called, like bt_gatt_init(). If not, bt_rand will crash with a
+	 * segmentation fault, as it will be internally using a uninitialized
+	 * pointer.
+	 */
+	int err;
+
+	err = bt_rand(buf, len);
+	atomic_dec(&DEV_DATA(device)->user_count);
```

Comment:
dont need this or the data declared in the beginning of the file.

Reply:
done